common/network-common/src/test/java/org/apache/spark/network/crypto/AuthEngineSuite.java

```diff
@@ -188,19 +186,6 @@ public void testMismatchedSecret() throws Exception {
     }
   }
 
-  @Test(expected = AssertionError.class)
```

Comment:
This UT before `Update Spark key negotiation protocol` as follows:

```scala
  @Test(expected = InvalidKeyException.class)
  public void testBadKeySize() throws Exception {
    Map<String, String> mconf = ImmutableMap.of("spark.network.crypto.keyLength", "42");
    TransportConf conf = new TransportConf("rpc", new MapConfigProvider(mconf));

    try (AuthEngine engine = new AuthEngine("appId", "secret", conf)) {
      engine.challenge();
      fail("Should have failed to create challenge message.");

      // Call close explicitly to make sure it's idempotent.
      engine.close();
    }
  }
```
however, `spark.network.crypto.keyLength` is no longer used now, so this UT is unnecessary